### ISSUE TITLE
1.13.5 fails to install in a node.js docker container

### ISSUE THREAD

**Issue by RomanKisilenko** [jashkenas/underscore#2967]
npm ERR! file sh
npm ERR! code ELIFECYCLE
npm ERR! errno ENOENT
npm ERR! syscall spawn
npm ERR! underscore@1.13.5 postinstall: `patch-package`
npm ERR! spawn ENOENT
npm ERR! 
npm ERR! Failed at the underscore@1.13.5 postinstall script.

Please revert the failed release or release a bugfix.

**Comment by vincent-ogury** [comment]
same issue here:
```
node_modules/underscore: Command failed.
Exit code: 127
Command: patch-package
Arguments:
Directory: node_modules/underscore
Output:
/bin/sh: patch-package: command not found
```

**Comment by Nauzer** [comment]
Same here on node v12.22.7

**Comment by rubenesda** [comment]
Same issue on node v11.14.0 Could this be fixed?

**Comment by vincent-ogury** [comment]
solve for now with:
```
  "resolutions": {
    "underscore": "1.13.4"
  },
```
in the package.json.
It will force underscore to this version

**Comment by SebastianBar** [comment]
@jgonggrijp a hand over here, please? It seems to be related to https://github.com/jashkenas/underscore/commit/08cb1404eb27b80bcadb87a32fcd515f23f7c9a6

**Comment by leithouse** [comment]
The issue is in the post install script in package.json
https://github.com/jashkenas/underscore/blob/68e5eb68635548846b4663479b9d0fc78d090b0a/package.json#L95

**Comment by nicolasKuhn-y** [comment]
Same Issue. It broke a popular app

**Comment by yannickcare** [comment]
> solve for now with:
> 
> ```
>   "resolutions": {
>     "underscore": "1.13.4"
>   },
> ```
> 
> in the package.json. It will force underscore to this version

I added this in my package.json, also the 
`  "scripts": {
    "preinstall": "npx npm-force-resolutions"
  },`

but I still getting the issue underscore@1.13.5 postinstall: `patch-package`

**Comment by yannickcare** [comment]
> > solve for now with:
> > ```
> >   "resolutions": {
> >     "underscore": "1.13.4"
> >   },
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > in the package.json. It will force underscore to this version
> 
> I added this in my package.json, also the ` "scripts": { "preinstall": "npx npm-force-resolutions" },`
> 
> but I still getting the issue underscore@1.13.5 postinstall: `patch-package`
> 
> Update: # npm WARN lifecycle c4g-api@0.0.1~preinstall: cannot run in wd c4g-api@0.0.1 npx force-resolutions (wd=/app)

Modified the DockerFile to include : 
`RUN npm install --production --unsafe-perm`

which let me run the "resolutions" but in the end I still have underscore@1.13.5 postinstall /app/node_modules/underscore

**Comment by Keysox** [comment]
Another workaround is `yarn install --ignore-scripts`

**Comment by coughlinj** [comment]
Workaround:

`npm i patch-package `

Before doing anything that requires underscore as a dependency.

**Comment by SantiagoMejorada** [comment]
> Workaround:
> 
> `npm i patch-package `
> 
> Before doing anything that requires underscore as a dependency.

This save my build! Thanks a lot!

**Comment by jgonggrijp** [comment]
I just released 1.13.6, hopefully that fixes it. Please close if that is the case.
